### ISSUE TITLE
Update WebP support

### ISSUE THREAD

**Issue by iJungleboy** [imazen/resizer#256]
There is an alpha/beta support for WebP but it uses a very old version of the `libwebp` library. 

We would like to deploy WebP (inside 2sxc for DNN - a very popular plugin running on ca. 20'000 DNN installations), but with such an old DLL (0.6) we're worried about possible security issues. 

I tried to just use a newer version of libwebp and was even able to compile it after messing around with visual studio, C compilers etc. but it's not compatible. 

Could you update your connector to the latest version of libwebp? Would really appreciate it. 

Many thanks! iJungleboy

**Comment by lilith** [comment]
Yes, this is in the backlog. Have you considered using [Imageflow.Net](https://github.com/imazen/imageflow-dotnet) instead? It uses the latest version of webp and supports much better jpeg compression as well.

**Comment by iJungleboy** [comment]
I've considered it - I'm/we (2sic) are one of your backers ;) and really excited to see progress again. 

ATM I'm still very hesitant to start. We are the makers of [2sxc](https://2sxc.org) - the most important CMS plugin for DNN, and we bundle ImageResizer with that. So switching resizers impacts somwhere between 10'000 and 25'000 websites which can disrupt stuff, + potentially cause quite some distress because of the AGPL nature of the ImageFlow.

So for my company 2sic it wouldn't be a problem - but it's hard to guess how all other users will be affected, which is why we're kind of scared to go that step. 
We're also not sure if it will be a breaking change, if non-native DLLs will work (I believe that it can cause issues when run from UNC-shares, etc). So we're extremely cautious, but will look into that as it stabilizes. 

That's why we're really hoping to get WebP to work asap. 

BTW: as of now, we really had issues running it in DNN where the files are running on a UNC share, also because of the C++ dependencies which are usually missing on a "pure" hosting server . I'll explain more or create an issue if you're interested.

**Comment by lilith** [comment]
I would definitely be interested in more info; please open an issue!

**Comment by iJungleboy** [comment]
Just created #257 and #258
